src/Education/Commands/RuleDescriptionUserControl.xaml

```diff
@@ -42,7 +42,14 @@
         </Style>
         <Style x:Key="SubtitleElement_Span" TargetType="Span">
             <Setter Property="FontSize" Value="{DynamicResource {x:Static vsshell:VsFonts.CaptionFontSizeKey}}"/>
-            <Setter Property="FontWeight" Value="{DynamicResource {x:Static vsshell:VsFonts.CaptionFontWeightKey}}"/>
+            <Setter Property="FontWeight" Value="Bold"/>
+        </Style>
+        <Style x:Key="SubtitleElement_Image" TargetType="Image">
+            <Setter Property="Margin" Value="0,0,3,0" />
+            <Setter Property="Stretch" Value="None" />
+            <!--<Setter Property="Stretch" Value="UniformToFill" />
+            <Setter Property="Height" Value="12" />
+            <Setter Property="Width" Value="12" />-->
```

Comment:
I added another style so we can control the appearance of the images from the XAML.
I left the properties setters for `Stretch`, `Height` and `Width` for now as a reminder of alternatives for controlling the image size in case it needs any tweaking.